packages/timezone_map/lib/src/geoip.dart

```diff
@@ -8,8 +8,8 @@ import 'location.dart';
 import 'source.dart';
 
 final _options = BaseOptions(
-  connectTimeout: 5000,
-  receiveTimeout: 5000,
+  connectTimeout: const Duration(seconds: 5000),
+  receiveTimeout: const Duration(seconds: 5000),
```

Comment:
these were probably in milliseconds before?
```suggestion
  connectTimeout: const Duration(seconds: 5),
  receiveTimeout: const Duration(seconds: 5),
```

Reply:
uh, very good point :sweat_smile: